android/src/main/java/so/onekey/app/wallet/lite/onekeyLite/nfc/Connection.kt

```
@@ -22,9 +23,14 @@ class Connection(val isoDep: IsoDep, private val mCommandGenerator: CommandGener
 
         @JvmStatic
         private fun connect(isoDep: IsoDep?) {
-            if (isoDep?.isConnected == false) {
-                isoDep.connect()
-                isoDep.timeout = 15000
+            val isConnected: Boolean = try {
+                isoDep?.isConnected ?: false
+            } catch (e: Exception) {
```

detekt

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)